### ISSUE TITLE
A lot of the examples seem to break the bounding div

### ISSUE THREAD

**Issue by lacymorrow** [STRML/textFit#25]
maybe just me...

chrome OSX newest of now

**Comment by alpha-and-omega** [comment]
Yes, I have seen it in other project. This bug is related to Safari webkit engine. And all div>span fitters have this bug.
It can be fixed using two subsequent span's.
